app/jinja_filters.py

```diff
@@ -350,6 +350,24 @@ def format_unordered_list(context, list_items):
     return mark_safe(context, summary_list)
 
 
+@evalcontextfunction
+def format_unordered_list_missing_items(context, possible_items, list_items):
+    summary_list = ''
+
+    if list_items and list_items[0]:
+        not_answered = [item for item in possible_items if item not in list_items[0]]
+    else:
+        not_answered = possible_items
+
+    if not_answered:
```

Comment:
Could we avoid some duplication here by having a filter to subtract a list from another list, and then pass that into `format_unordered_list`? Feel like it would make unit tests a little nicer too since we wouldn't be checking any HTML.

Obviously would make using the filter a little more complex though.

Reply:
Yes, I prefer that and I don't feel that it really complicates the filter.

Reply:
I didn't do a subtraction though as you'd lose the list order while using set().

Reply:
Ah, I actually meant having two separate filters and using them both in the schema. But this looks good.